docs/simple_switch.md

```diff
@@ -685,3 +685,12 @@ you have two actions for table `t` where one has the action
 `m.read(result_field2)`, or if both of those calls are in the same
 action.  All calls to the `read()` method for `m` must have the same
 result parameter where the result is written.
+
+### BMv2 `meter rate and burst_size units` implementation notes.
```

Comment:
Why is this between backticks?
why do you think this qualifies as "implementation notes"?

Reply:
I removed the backticks since the exact names of the CLI args are not used.  Also removed "implemented notes"